src/components/publicApi/LoginScreen.tsx

```diff
@@ -53,6 +54,8 @@ interface Props {
   onLogin: OnLogin
   // Called when the user makes a choice from RequestPermissionsModal:
   onNotificationPermit?: OnNotificationPermit
+  // Passed from the GUI for analytics reporting
+  onLogEvent: OnLogEvent
```

Comment:
These need to be optional otherwise we have a breaking change to the login-ui API